pkg/components/metallb/component_test.go

```diff
@@ -186,14 +179,17 @@ component "metallb" {
 		t.Fatalf("Rendered manifests shouldn't be empty")
 	}
 
+	ds := getSpeakerDaemonset(t, m)
+	deploy := getDeployController(t, m)
+
 	tcs := []struct {
 		Name string
 		Test func(*testing.T, map[string]string)
 	}{
 		{
 			"SpeakerConversions", func(t *testing.T, m map[string]string) {
-				ds := getSpeakerDaemonset(t, m)
 				expected := []corev1.Toleration{
+					{Key: "node-role.kubernetes.io/master", Operator: "Exists", Effect: "NoSchedule"},
```

Comment:
What does this toleration do ?
In Lokomotive we have `node.kubernetes.io/master` as the label for controller.

Reply:
It's a built-in from chart and by default the chart appends user tolerations to default ones?

Reply:
ok

Reply:
Perhaps we should disable those instead, to avoid confusion :)

Reply:
should it be scope of different PR right?

Reply:
It seems like a side-effect of this PR, so I'd say no

Reply:
But it seems speaker tolerations are the only side-effect, so it's not a lot of changing.